src/IB/IBFEMethod.cpp

```diff
@@ -1017,6 +1022,41 @@ IBFEMethod::spreadForce(const int f_data_idx,
         d_ghost_data_accumulator->accumulateGhostData(f_scratch_data_idx);
     }
 
+    // Prolong forces spread onto coarser levels onto finer levels.
+    if (d_use_scratch_hierarchy)
+    {
+        // not yet implemented
+        assertStructureOnFinestLevel();
+    }
+    else
+    {
+        // TODO - we should save these schedules in the same manner that we save
+        // the schedules for the scratch hierarchy.
+        for (int ln = getCoarsestPatchLevelNumber(); ln < getFinestPatchLevelNumber(); ++ln)
+        {
+            Pointer<RefineAlgorithm<NDIM> > refine_algorithm = new RefineAlgorithm<NDIM>();
+            Pointer<CartesianGridGeometry<NDIM> > geometry = d_hierarchy->getGridGeometry();
+            // TODO: this is probably the wrong refinement operator
+            Pointer<RefineOperator<NDIM> > refine_op =
+                geometry->lookupRefineOperator(f_var, "CONSERVATIVE_LINEAR_REFINE");
+            // TODO: do we need another scratch data index? We should be able to
+            // prolong directly into the output index
+            refine_algorithm->registerRefine(
+                f_prolong_scratch_data_idx, f_scratch_data_idx, f_prolong_scratch_data_idx, refine_op);
+            Pointer<PatchLevel<NDIM> > coarse_level = d_hierarchy->getPatchLevel(ln);
+            Pointer<PatchLevel<NDIM> > fine_level = d_hierarchy->getPatchLevel(ln + 1);
+            // We can ignore the fourth argument since we don't need to read
+            // forces outside the physical domain
+            Pointer<RefineSchedule<NDIM> > schedule = refine_algorithm->createSchedule(
+                "FILL_INTERIORS_ONLY", fine_level, coarse_level, nullptr, false, nullptr);
+            schedule->fillData(data_time);
```

Comment:
I would do this all in `FEDataManager`...

Reply:
...and @drwells and I just discussed on Slack why this is not a good thing to do for multi-part models.

Reply:
In summary - we don't want to communicate inside FEDataManager when looping over parts since it leads to big workload imbalances.

Reply:
It might be good someday to have something that sits between `IBFEMethod` and the single-part `FEDataManager` so that `IBFEMethod` doesn't need to know so much about what `FEDataManager` does.

Reply:
They have a very unhealthy codependent relationship :( see also #715.